### DISCUSSION TITLE
PHP 7.4.12, curl-openssl 7.73 and Apache

### DISCUSSION THREAD

**Discussion by gaspardphilibert** [Homebrew/discussions#123] [Everyday usage] [answered]
Hi !

I've been using Homebrew's PHP with Apache (macOS version) for years. But with PHP 7.4.12 and curl-openssl 7.73, Apache doesn't want to start anymore. I tried reverting to macOS-bundled PHP in httpd.conf and it worked. 
I looked in crash reports and found that there was a problem with /usr/local/opt/curl-openssl/lib/libcurl.4.dylib. As I wanted PHP 7.4, I went back to PHP 7.4.11 and an older version of curl-openssl and Apache works again.
I wonder if someone encounters this issue with the new curl(-openssl) dylib ?

Thanks
PS : I'm on macOS 10.15.7

**Comment by SMillerDev** [comment, marked as answer]
built-in Apache isn't supported with the Homebrew PHP. As you noticed it breaks very easily and Homebrew can't really test the compatibility. The best solution would be to switch to the httpd version that Homebrew ships. The second best solution would be to report a solid testcase to Apple or PHP so they can debug this.
